README.md

```diff
@@ -48,28 +47,35 @@ Please ensure that you have at least the **minimum** recommended versions
 #### 2. Install packages
 
 -   Install the packages
+
+    ```bash
+    yarn install
```

Comment:
Just pick one to document, don't make an unfamiliar user have to think/research the difference :)

Reply:
Cool. I will pick `yarn install` because its more specific. If anyone else feels otherwise, please let me know!